lib/internal/modules/esm/translators.js

```diff
@@ -27,10 +27,18 @@ module.exports = translators;
 translators.set('esm', async (url) => {
   const source = `${await readFileAsync(new URL(url))}`;
   debug(`Translating StandardModule ${url}`);
-  return {
-    module: new ModuleWrap(stripShebang(source), url),
-    reflect: undefined
-  };
+  const module = new ModuleWrap(stripShebang(source), url);
+  const {
+    initializeImportMetaMap,
+    importModuleDynamicallyMap
+  } = require('internal/process/esm_loader');
```

Comment:
Is this an issue with circular references or something? Surely it's already in the graph at this point that it can be hoisted?